guides/common/modules/proc_installing-the-load-balancer.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Installing_the_Load_Balancer_{context}"]
 = Installing the load balancer
 
-The following example provides general guidance for configuring an HAProxy load balancer using {EL} 9 or {EL} 8.
+The following example provides general guidance for configuring an HAProxy load balancer using {EL} 9, {EL} 8, or {EL} 7.
```

Comment:
I don't think we should include EL7 here, as this is purely an example, and we want to push users to latest and greatest wherever possible. I think what we already have in this guide is sufficient and no other work is necessary.

Reply:
Can we keep upto EL 8?